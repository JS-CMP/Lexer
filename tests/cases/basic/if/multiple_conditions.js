var x = 5, y = 10;
if (x < 10 && y > 5) {
    console.log("Both conditions are true");
}