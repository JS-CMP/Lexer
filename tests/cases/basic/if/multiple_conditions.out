#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {JS::Any x, y;x=JS::Any(5);y=JS::Any(10);if (x<JS::Any(10)&&y>JS::Any(5)){console[u"log"](JS::Any(u"Both conditions are true"));};}
